index.html

```diff
@@ -853,10 +871,10 @@ <h4>publicKeyBase58</h4>
           <tbody>
             <tr>
               <td>
-                <a href="https://w3id.org/security/#publicKeyBase58">security-vocab</a>
+                <a href="https://w3id.org/security/#publicKeyBase58">security-vocab V2</a>
```

Comment:
This is probably why my grammar isn't very good. They re-taught us how to write a 3 paragraph essay for 3 grade levels rather than slapping our wrists with rulers when we got our punctuation wrong 🙃 That combined with me going into computer science because I didn't want to write long papers in college and now you and @TallTed end up begrudgingly end up being a spell check rather than a content editor so we can make long standing, useful documents.

Oh how the education systems have changes. In any case it's much appreciated to have your reviews and I'll be more careful when adding to specs to make the review experience easier on others which is arguably the harder part of this process.